floodlight/io/catapultApi.py

```diff
@@ -31,63 +46,63 @@ def saveResponse(response: requests.Response, endpoint: str) -> None:
     else:
         print(f"Error: {response.status_code} - {response.text}")
 
-
-def load_responses(endpoint: str) -> List[Dict]:
+def read_response(endpoint: str) -> List[Dict]:
     """
-    Loads and returns a list of JSON data responses from the specified directory.
-
-    Args:
-        endpoint (str): The directory containing the JSON files.
-
-    Returns:
-        List[Dict]: A list of JSON data responses.
-
-    Raises:
-        FileNotFoundError: If the specified directory does not exist.
+    Loads and returns a list of response_data from the specified directory.
+
+    Parameters
+    ----------
+    endpoint : str
+        The directory containing the JSON files.
+
+    Returns
+    -------
+    List[Dict]
+        A list of response_data.
+
+    Raises
+    ------
+    FileNotFoundError
+        If the specified directory does not exist.
     """
-    response_data_json_list = []
+    response_data = []
     
-    # Check if the endpoint directory exists
     if os.path.exists(endpoint):
-        # Walk through all directories and files in the given directory
         for root, dirs, files in os.walk(endpoint):

```

Comment:
I may be missing something but wouldn't `os.listdir(endpoint)` be sufficient here?

Reply:
yes it is it is changced accordingly